src/classes/SearchMessager.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Search Messager
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import ExtMessenger from './ExtMessenger';
+import account from './Account';
+import { log } from '../utils/common';
+
+/**
+ * Class for handling cross-extension messaging.
```

Comment:
Can you throw a @since into the JSDoc comment?

Reply:
....What the heck is going on with the @ since formatting, ?lol?

Reply:
You tagged that github user. I updated the comment.